src/test/unit/test_w3c_context.py

```diff
@@ -0,0 +1,80 @@
+import pytest
```

Comment:
If I understood correctly, the code was copied from OTL, was the test file also copied? If so, please add a comment.